js/apps/admin-ui/src/user-federation/ldap/LdapSettingsAdvanced.tsx

```diff
@@ -4,7 +4,7 @@ import { useTranslation } from "react-i18next";
 
 import { useAlerts } from "../../components/alert/Alerts";
 import { FormAccess } from "../../components/form-access/FormAccess";
-import { HelpItem } from "ui-shared";
+import { HelpItem } from "../../components/help-enabler/HelpItem";
```

Comment:
Why is this no longer imported from `ui-shared`?

Reply:
Hi! I don't know for sure, I just imported the files from my commit to the former ui-repo. Maybe the new version overwrites the previous and this gets removed?

Reply:
@jonkoops I can put it back if you want. Just say the word.

Reply:
yes it would be better to put that back

Reply:
Yes, we do prefer using the components from `ui-shared` where possible.